src/encoded/schemas/analysis.json

```diff
@@ -0,0 +1,96 @@
+{
+    "title": "Analysis",
+    "description": "Schema for grouping files into one analysis.",
+    "comment": "An analysis is typically performed on one experiments and is initially designed to group only files from one same experiment.",
+    "id": "/profiles/analysis.json",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "required": ["dataset", "files"],
```

Comment:
I am not 100% dataset should be required.  I am concerned that if/when we create analyses that span multiple datasets this will be hard to port.